examples/nextjs/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "@primer/octicons-react": "^18.2.0",
-    "@primer/react": "file:../../",
+    "@primer/react": "^35.21.1",
```

Comment:
I am not sure if I am reading this right so let me know if I misunderstand this change. 

I rely on so much building the react locally and immediately make it available to the nextjs application so that I can debug and resolve issues as I go. As far as I understand keeping a stable version here won't allow this? If I am reading this right, is there a way to keep this practicality and convenience?

Reply:
>This version here will automatically resolve to the packages/react folder because node_modules/@primer/react points to that workspace.

🤯  It might be just me but I wouldn't assume that it will resolve. The version number misled me. Thanks for explaining that.